homework_1/task_1/fizzbuzz/fizzbuzz_test.go

```diff
@@ -0,0 +1,31 @@
+package fizzbuzz_test
+
+import (
+	"fmt"
+	. "github.com/smartystreets/goconvey/convey"
+	"task1/fizzbuzz"
+	"testing"
+)
```

Comment:
Usually we want to separate imports by their type into groups with newlines between them. The three groups we choose usually are:
* standard library imports (in this case `fmt`, `testing`)
* third party libraries (in this case `github.com/smartystreets/goconvey/convey`)
* this repository (in this case `task1/fizzbuzz`)

This really comes in handy when there are files with tens of imports. It can get cluttered really quickly otherwise.
So I would suggest changing this to 
```suggestion
import (
	"fmt"
	"testing"
	
	. "github.com/smartystreets/goconvey/convey"
	
	"task1/fizzbuzz"
)
```